docs/guides/cloud/introduction.mdx

```diff
@@ -73,17 +75,18 @@ the latest results across the team, and identify areas of concern.
 
 ### Run tests in parallel, in priority order, or not at all
 
-With our [Smart Orchestration](/guides/cloud/smart-orchestration) features, you
-can run tests across a swarm of machines simultaneously while Cypress Cloud
-coordinates runners and
-[balances test loads](/guides/guides/parallelization#Balance-strategy) - no
-setup required! You can prioritize recently failed specs with
-[Spec Prioritization](/guides/cloud/smart-orchestration#Spec-Prioritization) to
+With our
+[Smart Orchestration](/guides/cloud/smart-orchestration/parallelization)
+features, you can run tests across a swarm of machines simultaneously while
+Cypress Cloud coordinates runners and
+[balances test loads](/guides/cloud/smart-orchestration/load-balancing#Balance-strategy) -
+no setup required! You can prioritize recently failed specs with
+[Spec Prioritization](/guides/cloud/smart-orchestration/spec-prioritization) to
 surface problems earlier, and cancel whole test runs on failure with
-[Auto Cancellation](/guides/cloud/smart-orchestration#Auto-Cancellation) to save
+[Auto Cancellation](/guides/cloud/smart-orchestration/run-cancellation) to save
 on resource usage. You can also
-[cancel in-progress runs](/guides/cloud/runs#Cancel-run) manually from Cypress
-Cloud if you need to.
+[cancel in-progress runs](/guides/cloud/recorded-runs#Run-cancellation) manually
```

Comment:
I would expect to find this content on the runs cancellation page. Is there a plan to condense this to live in the same spot relatively soon?

Reply:
https://github.com/orgs/cypress-io/projects/16/views/11?pane=issue&itemId=30695406